gui/wxpython/rdigit/toolbars.py

```diff
@@ -195,3 +195,27 @@ def _widthValueChanged(self):
     def _changeDrawColor(self):
         color = self._color.GetColour()
         self._controller.ChangeDrawColor(color=color)
+
+    def MapSelectionEvt(self, action='unbind'):
+        """Unbind and bind map selection combobox widget event
+
+        :param action str: 'unbind' or 'bind' event handler
+        """
+        if action == 'unbind':
+            self._mapSelectionCombo.Unbind(wx.EVT_COMBOBOX)
+        else:
+            self._mapSelectionCombo.Bind(
+                wx.EVT_COMBOBOX, self.OnMapSelection,
+            )
+
+    def SetMap(self, mapName):
```

Comment:
Why not call `OnMapSelection` directly? It could be split into `MapSelected`:
```
def OnMapSelection(self, event):
    self.MapSelected()
```
and then MapSelected would be called after we programmatically change the value. Or is there a reason to do it the way you suggest?

Reply:
The method event `OnMapSelection()` is now called [directly](https://github.com/OSGeo/grass/commit/691bf4eed96e672ec5190f9a29927288b62b5be3#diff-be3d2463d0f27254225a23d5a4427682b7fc9efd4b0aeb283ecc3dd204323514R116).